File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyListOfImagesRow.cs
/// <filename>
///     ListViews_ViewModelMyListOfImagesRow.cs
/// </filename>


namespace widgetassertions
{
    public class ListViews_ViewModelMyListOfImagesRow
  {
        private string MyInfoImageName;
     
        public     virtual string getMyInfoImageName()
    {
      return this.MyInfoImageName;
    }
     
        private bool isMyInfoImageEnabled;
     
        public     virtual bool getIsMyInfoImageEnabled()
    {
      return this.isMyInfoImageEnabled;
    }
     
        private bool isMyInfoImageVisible;
     
        public     virtual bool getIsMyInfoImageVisible()
    {
      return this.isMyInfoImageVisible;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyListOfImagesRow.cpp

#include "ListViews_ViewModelMyListOfImagesRow.h"
#include <string>

namespace widgetassertions
{
  std::string ListViews_ViewModelMyListOfImagesRow::getMyInfoImageName( ) 
  {
    return this->MyInfoImageName;
  }
  bool ListViews_ViewModelMyListOfImagesRow::getIsMyInfoImageEnabled( ) 
  {
    return this->isMyInfoImageEnabled;
  }
  bool ListViews_ViewModelMyListOfImagesRow::getIsMyInfoImageVisible( ) 
  {
    return this->isMyInfoImageVisible;
  }
  int ListViews_ViewModelMyListOfImagesRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

